pandas/core/frame.py

```diff
@@ -5432,7 +5433,7 @@ def _put_str(s, space):
 
 import pandas.tools.plotting as gfx
 
-DataFrame.plot = gfx.plot_frame
+DataFrame.plot = base.AccessorProperty(gfx.FramePlotMethods, gfx.FramePlotMethods)
 DataFrame.hist = gfx.hist_frame
```

Comment:
The defaults on `DataFrame.hist` are different that `DataFrame.plot.hist` which I think is why we have both for now. I could go either way on deprecating `DataFrame.hist`.